.github/workflows/docs-pull-requests.yaml

```diff
@@ -9,8 +9,8 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
-    lint:
-        name: Lint
+    storybook:
```

Comment:
Why is it needed?

Reply:
to ensure taht the preview build is not failing. else fondue may not be accessible and we will not know